Topics/Development_Process/Duckdns.md

```diff
@@ -0,0 +1,53 @@
+# Overcoming a Dynamic IP Address with DuckDNS
+
+## Motivation
```

Comment:
Wow, the Motivation is just superb! It got me hooked as I've actually faced this issue myself when I shutdown my EC2 instance just to find out the IP _changed_ the following time when I rebooted the instance causing a big pain as I had to change the references to to that IP in my code.